src/adapters/livekit.ts

```diff
@@ -1,29 +1,20 @@
-import { IBaseComponent } from '@well-known-components/interfaces'
-import { AppComponents, LivekitCredentials, Permissions } from '../types'
-import { AccessToken, RoomServiceClient, TrackSource } from 'livekit-server-sdk'
-
-export type LivekitSettings = {
-  host: string
-  apiKey: string
-  secret: string
-}
-
-export type ILivekitComponent = IBaseComponent & {
-  generateCredentials: (
-    identity: string,
-    roomId: string,
-    permissions: Omit<Permissions, 'mute'>,
-    forPreview: boolean
-  ) => Promise<LivekitCredentials>
-  muteParticipant: (roomId: string, participantId: string) => Promise<void>
-  getWorldRoomName: (worldName: string) => string
-  getSceneRoomName: (realmName: string, sceneId: string) => string
-}
+import { AppComponents, ILivekitComponent, LivekitCredentials, LivekitSettings, Permissions } from '../types'
+import {
+  AccessToken,
+  CreateIngressOptions,
+  IngressClient,
+  Room,
+  RoomServiceClient,
+  TrackSource
+} from 'livekit-server-sdk'
+import { IngressInfo, IngressInput } from 'livekit-server-sdk/dist/proto/livekit_ingress'
 
 export async function createLivekitComponent(
   components: Pick<AppComponents, 'config' | 'logs'>
 ): Promise<ILivekitComponent> {
-  const { config } = components
+  const { config, logs } = components
```

Comment:
We should consider adding unit tests to this component, as there are new methods being implemented and other that changed